### ISSUE TITLE
Deprecated gRPC insecure usage

### ISSUE THREAD

**Issue by sideninja** [onflow/flow-cli#430]
### Problem
Usage of grpc insecure option is deprecated and should be fixed:
```bash
pkg/flowkit/gateway/grpc.go:49:3: SA1019: grpc.WithInsecure is deprecated: use insecure.NewCredentials() instead. Will be supported throughout 1.x. (staticcheck)
		grpc.WithInsecure(),
		^
```

However this opens up more questions, should we adapt Go SDK to handle this or actually think about moving to a secure connection by default.

**Comment by sideninja** [comment]
A temporary workaround is to use `insecure.NewCredentials()` but we should discuss secure transport and usage of certificates.

**Comment by sideninja** [comment]
Closed with #449
